### PR TITLE
Subcompany compartment

### PR DESCRIPTION
**Pull request by Neamar**

### CODE REVIEW

File: blueprint.md [hunk 1]
```diff
@@ -248,6 +248,8 @@ You may customize hydraters list, or use default list.
 
 The user executing the call (either via `Basic` or `Bearer`) will be migrated to the new company. To create a new subcompany, the best practice is to create a new admin (`POST /users`) before, then create the new company while connected with the new user.
 
+Note the "original" company will only be able to `DELETE` the subcompany, and can't do anything more: the new user will effectively be the admin in the subcompany, and the only one able to view and add documents.
```
Comment (merlinND) [inline blueprint.md]: Technically, the admin will not be the _only one_... He can create new users in the subcompany.

Comment (Neamar) [inline blueprint.md]: and the only one able to view and add documents _and users_

Would that be better ?

Comment (merlinND) [inline blueprint.md]: and the only one able to view and add documents, users, ...

**Comment by merlinND** [comment]
GTM

**Pull request merged**